frontend/src/pages/LandlordPage.tsx

```diff
@@ -48,18 +54,21 @@ const LandlordPage = (): ReactElement => {
   useEffect(() => {
     get<ReviewWithId[]>(`/review/landlordId/${landlordId}`, {
       callback: setReviewData,
+      errorHandler: handlePageNotFound,
```

Comment:
Same thing here- the error handler shouldn't be used for the review or buildings useEffect.